src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -1,12 +1,27 @@
 package racingcar.domain
 
-class Car {
-    var position: Int = 0
+class Car(
+    val name: String,
+    position: Int = INIT_POSITION
+) {
+    var position: Int = position
         private set
```

Comment:
```suggestion
class Car(
    val name: String
) {
    var position: Int = INIT_POSITION
        private set
```

생성자의 position은 없어도 될 것 같아요!